[apache-couchdb] Add github repo url ,  fix versionCommand

adriens opened the pull request:
While discovering this [tweet](https://x.com/TheASF/status/1989751534964375569)

<img width="590" height="611" alt="image" src="https://github.com/user-attachments/assets/bb0b1db2-d781-4aa3-bff5-a5e8bea8add6" />

I discovered that this product was missing the gtihub repo.

I didn't really know how it should be added, let me know what you thing

adriens committed:
`apache-couchdb`  > Added gh repo url

adriens committed:
Update apache-couchdb.md

adriens commented:
<img width="889" height="343" alt="image" src="https://github.com/user-attachments/assets/008417eb-07ff-42e8-b8d7-e564cea7ae61" />

adriens committed:
Update apache-couchdb.md

Update version command

adriens commented:
> Does jq exist every system directly ? if might be not so suitable

You have a point here @usta ... let me try with a standard tool :+1:

adriens committed:
Merge branch 'master' into patch-1

adriens committed:
Update apache-couchdb.md

cf
- https://github.com/endoflife-date/endoflife.date/pull/8955#issuecomment-3537164575

usta commented:
I'm reverting my claim ;) This looks much worse :D users might install jq :D
[sorry to make you spend time]

adriens commented:
> I'm reverting my claim ;) This looks much worse :D users might install jq :D
> [sorry to make you spend time]

Indeed yes it looks awful....do you want the awk version? 

The jq is the cleanest anyway

usta commented:
> > I'm reverting my claim ;) This looks much worse :D users might install jq :D
> > [sorry to make you spend time]
> 
> Indeed yes it looks awful....do you want the awk version?
> 
> The jq is the cleanest anyway

I'm not here to make users suffer :D jq is better choice in here

adriens committed:
couchdb > get version, revert to curl  | jq

cf
- https://github.com/endoflife-date/endoflife.date/pull/8955#issuecomment-3538041555

adriens committed:
Merge branch 'master' into patch-1

usta commented:
@adriens I'm removing nosql tag ( to overcome problem that any tag couldnt have just one product rule)
if you want nosql tag in this , could you create another PR that will add nosql tag for other products too ( i mean mongo , ... )

usta committed:
Update tags for Apache CouchDB

Removed 'nosql' tag from Apache CouchDB metadata.
The rule is if a tag exist it should have multiple product in that tag.

adriens commented:
> LGTM , as like i mention you are free to create a seperate PR which includes multiple suitable products and add nosql to all in one in that PR.

yes, I wanted to knwo if you guys would be ok to implement this tag. i'll make a dedicated PR for that :+1:

adriens commented:
> LGTM , as like i mention you are free to create a seperate PR which includes multiple suitable products and add nosql to all in one in that PR.

- https://github.com/endoflife-date/endoflife.date/pull/8963

captn3m0 commented:
Late here, but don't see a point in adding github refs on the description. The description is for users of the product, and should link to the primary website. Plus we already have a link via identifiers to GitHub anyway.
